### DISCUSSION TITLE
When i `make native` no jar is generated. just a folder.

### DISCUSSION THREAD

**Discussion by ColtonIdle** [xerial/sqlite-jdbc#1036] [Q&A]
im trying to make a jar by building latest master from source. it seems successful. and it generates a new directory in `target/` but theres no jar. am i missing something? im a noob at make/jars  =)

**Comment by gotson** [comment]
`make native` only builds the native library, not the jar. You need to use Maven to build the jar.

**Reply by ColtonIdle** [reply]
What's the maven command? I installed maven as per the docs, but it didn't say anything about calling `maven`. let me try to chatgpt it.
